models/recipe.model.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const { User, UserSchema } = require("./user.model");
+
+const TagSchema = new Schema({
+    id: {
+        type: Number,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+});
+
+module.exports = TagSchema;
+module.exports = Tag = mongoose.model("tag", TagSchema);
+
+const IngredientSchema = new Schema({
+    id: {
+        type: Number,
```

Comment:
is there a reason why the id field in this schema is a number and in others it is type "mongoose.Schema.Types.ObjectId"?